.bufstyle.yaml

```diff
@@ -0,0 +1,4 @@
+ignore:
+  - path: private/bufpkg/bufstyle/cmd/bufstyle/main.go
+    analyzers:
+      - NO_SYNC_POOL
```

Comment:
I think we talked about this before, but what are your thoughts on inverting the control here? So we'd ignore by `analyzer` name, and include one or more paths to ignore.

```
ignore:
  - analyzer: NO_SYNC_POOL
     paths:
       - private/bufpkg/bufstyle/cmd/bufstyle/main.go
```

You could even reduce it down further to look like the `ignore_only` `lint` configuration:

```
ignore:
  NO_SYNC_POOL:
     - private/bufpkg/bufstyle/cmd/bufstyle/main.go
```

And if we ever wanted to make this more widely available, we could reserve `ignore` for excluding certain linters (like `lint.ignore`), and rename the configuration above to `ignore_only`, but we shouldn't optimize for that yet.

Reply:
I updated to this style for now:

```
ignore:
  NO_SYNC_POOL:
     - private/bufpkg/bufstyle/cmd/bufstyle/main.go
 ```